### DISCUSSION TITLE
How to work with different databases in ASReview?

### DISCUSSION THREAD

**Discussion by Mat0sky** [asreview/asreview#1197] [Q&A]
ASReview allows you to upload a search export document (e.g., RIS format in WOS), but if I want to compare the results with search export documents from other databases (e.g., Scopus, PsycInfo) how can I do it?

I can only work with the RIS exported from one database... It's possible to integrate more databases in the same analysis? Or should I do it manually and after removing duplicates upload the final document to ASReview?

Thank you!

**Comment by jteijema** [comment]
Hi @Mat0sky! 

ASReview uses a single dataset at a time. I would combine the datasets first and then upload the document to ASReview, as you suggest. 

It doesn't have to be all manual work though! Take a look at [ASReview Datatools](https://github.com/asreview/asreview-datatools) for converting file formats and deduplication. Only combining the datasets is not integrated in datatools yet.

Jelle

**Reply by J535D165** [reply]
We aren't planning to build a feature to combine datasets in ASReview LAB from various sources in the near future. The best solution would be to use a reference manager. You can use a free and open reference manager like Zotero, or proprietary software like EndNote. Both support reference management as well as deduplication. 

I would also like to refer to this discussion: https://github.com/asreview/asreview/discussions/574
